.github/workflows/ci.yml

```diff
@@ -35,31 +35,44 @@ jobs:
   htmlproofer:
     name: HTMLProofer
     runs-on: ubuntu-latest
-    
+
     steps:
       - name: Checkout EIP Repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-      
-      - name: Install OpenSSL
```

Comment:
Do we no longer need OpenSSL?

Reply:
No, we don't. Not sure why it was there in the first place.

Reply:
SG, consider adding "also remove OpenSSL" to PR title?